web_copy_confirm/static/src/js/web_copy_confirm.esm.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+import {patch} from "web.utils";
+import {ConfirmationDialog} from "@web/core/confirmation_dialog/confirmation_dialog";
+import {_t} from "@web/core/l10n/translation";
+import {FormController} from "@web/views/form/form_controller";
+
+patch(FormController.prototype, "user_recent_log/static/src/js/record_save.js", {
```

Comment:
Change the 'patchName'

Reply:
@Tardo Changed the patch name, Kindly review it. Thank you